### DISCUSSION TITLE
Freetube stopped working with Android

### DISCUSSION THREAD

**Discussion by Ronfearn** [FreeTubeApp/FreeTube#5699] [Q&A]
Freetube used to work on my Samsung A51 Android phone. Now, when I attempt to play a video or launch a music track,  it hangs up (I get a continuous rotating blue circle). Can anyone help? Thanks, Ronfearn

**Comment by CamilleHbp** [comment]
There is no FreeTube Android version. The application that you are mentioning is [this one](https://play.google.com/store/apps/details?id=com.freetube.live&hl=fr), I guess. It is not related to this project, they just used the same name, as far as I can understand. Especially considering this application sells ads, which is antithetical to Freetube's goal. Someone from the project could confirm.

What you want is [NewPipe](https://newpipe.net/). But you can't sync between NewPipe and FreeTube — yet.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@Ronfearn are you using https://github.com/MarmadileManteater/FreeTubeAndroid?

**Reply by Ronfearn** [reply]
I am using FreeTube version 2.5.0-PRODUCTION. At least that is what it says under About -> Version.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Where did you download this from?

**Reply by CamilleHbp** [reply]
Oh, I did not know about this project! Sorry ^^

**Comment by Ronfearn** [comment]
From Google Play Store. See screenshot.
Am I to understand that this App is fake FreeTube?
I have the same app on my notebook, and It is working fine.

Sent from Proton Mail Android

-------- Original Message --------
On 15/09/2024 21:05, efb4f5ff-1298-471a-8973-3d47447115dc wrote:
Where did you download this from?

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Correct this Application isnt affiliated with the project.

**Comment by Ronfearn** [comment]
Thanks very much for the information 😀

Sent from Proton Mail Android

-------- Original Message --------
